crates/bevy_dev_tools/src/fps_overlay.rs

```diff
@@ -59,21 +78,98 @@ impl Default for FpsOverlayConfig {
                 font_size: 32.0,
                 color: Color::WHITE,
             },
+            // TODO set this to display refresh rate if possible
+            frame_time_graph_config: FrameTimeGraphConfig::target_fps(60.0),
+        }
+    }
+}
+
+/// Configuration of the frame time graph
+#[derive(Clone, Copy)]
+pub struct FrameTimeGraphConfig {
+    /// Is the graph visible
+    pub enabled: bool,
+    /// The minimum acceptable FPS
+    ///
+    /// Anything bellow this will show a red bar
```

Comment:
```suggestion
    /// Anything below this will show a red bar
```

Reply:
Thanks, I make that mistake way too frequently 😅